src/main/java/com/google/research/bleth/simulator/Simulation.java

```diff
@@ -1,9 +1,399 @@
 package com.google.research.bleth.simulator;
 
+import com.google.common.collect.ImmutableList;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * An abstract class representing a BLETH simulation.
+ * Can be either a Tracing simulation or a Stalking simulation.
+ */
 public abstract class Simulation {
```

Comment:
As this is a pure abstract class, consider renaming to AbstractSimulation for clarity.